### PR TITLE
TAP 1.4.0 Doc edits

### PR DESCRIPTION
**Pull request by satya-dillikar**
*Issue #, if available:*
New TAP Release

*Description of changes:*
TAP 1.4.0 Doc edits for MultiCluster 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/deployment_guide/partner_editable/post_deployment.adoc [hunk 1]
```diff
@@ -145,8 +150,8 @@
 
 |`KubernetesVersion`
 |Kubernetes version
-|`1.22`
-|Version of Kubernetes control plane to deploy. The selected version must support the selected version of {partner-product-acronym}. For more information, refer to https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-prerequisites.html#kubernetes-cluster-requirements-3[Kubernetes cluster requirements^].
+|`1.23`
+|Version of Kubernetes control plane to deploy. The selected version must support the selected version of {partner-product-acronym}. For more information, refer to  https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/prerequisites.html#kubernetes-cluster-requirements-3[Kubernetes cluster requirements^].
```
Comment (tlindsay42) [inline docs/deployment_guide/partner_editable/post_deployment.adoc:154]:
```suggestion
|Version of Kubernetes control plane to deploy. The selected version must support the selected version of {partner-product-acronym}. For more information, refer to https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/prerequisites.html#kubernetes-cluster-requirements-3[Kubernetes cluster requirements^].
```

File: docs/deployment_guide/partner_editable/pre_deployment.adoc [hunk 2]
```diff
@@ -51,6 +54,9 @@ If necessary, request https://console.aws.amazon.com/servicequotas/home/[service
 
 |Route 53 private hosted zones
 |1
+
+|Elastic IP
```
Comment (tlindsay42) [inline docs/deployment_guide/partner_editable/pre_deployment.adoc:58]:
```suggestion
|Elastic IPs
```

**Review by tlindsay42 — COMMENTED** [review]
lgtm w/ a few suggestions

**Review by annaone — APPROVED** [review]

**Comment by tlindsay42** [comment]
+1 thanks @annaone!

**Pull request merged**